File: libnd4j/include/dsp/runtime/bindings/kotlin/examples/BasicUsage.kt
import org.nd4j.dsp.runtime.KotlinSdxRuntime

fun main() {
    KotlinSdxRuntime.create().use { runtime ->
        println("ABI version: ${runtime.abiVersion()}")

        runtime.loadModel("path/to/model.sdx").use { model ->
            model.createContext().use { ctx ->
                println("Runtime created successfully.")
            }
        }
    }
}

File: libnd4j/include/dsp/runtime/bindings/rust/examples/basic_usage.rs
use sdx_runtime::{runtime_abi_version, Runtime};

fn main() {
    println!("ABI version: {}", runtime_abi_version());

    let runtime = Runtime::create(None).expect("Failed to create runtime");
    let _model = runtime
        .load_model("path/to/model.sdx", None)
        .expect("Failed to load model");

    println!("Runtime created successfully.");
}
